BoxOffice/Controller/MovieInfoViewController.swift

```diff
@@ -0,0 +1,200 @@
+                    self?.fetchMoviePoster()
+                }
+            case .failure(let error):
+                DispatchQueue.main.async {
+                    self?.displayAlert(from: error)
+                }
+            }
+        }
+    }
+    
+    private func fetchMoviePoster() {
+        var api = BoxOfficeURLRequest(service: .moviePoster)
+        let queryName = "query"
+        let movieName = movie?.result.movieInfo.movieName ?? ""
+        
+        api.addQuery(name: queryName, value: (movieName + " 영화 포스터"))
+        
+        let urlRequest = api.request()
+        
+        networkManager.fetchData(urlRequest: urlRequest, type: MoviePoster.self) { [weak self] result in
+            switch result {
+            case .success(let data):
+                self?.moviePoster = data
+                
+                guard let url = URL(string: data.documents.first?.imageURL ?? ""),
+                      let data = try? Data(contentsOf: url) else { return }
+                
+                DispatchQueue.main.async {
+                    self?.posterImageView.image = UIImage(data: data)
+                    self?.loadingView.isLoading = false
+                }
+            case .failure(let error):
+                DispatchQueue.main.async {
+                    self?.displayAlert(from: error)
+                }
+            }
+        }
+    }
+    
+    private func displayAlert(from error: Error) {
+        guard let networkingError = error as? NetworkingError else { return }
+        let alert = UIAlertController(title: networkingError.description, message: "모리스티에게 문의해 주세요.", preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "닫기", style: .cancel))
+        present(alert, animated: true)
+    }
+    
+    private func configureScrollView() {
+        view.addSubview(contentScrollView)
+        view.addSubview(loadingView)
+        
+        contentScrollView.addSubview(posterImageView)
+        contentScrollView.addSubview(movieInfoListStackView)
+        
+        NSLayoutConstraint.activate([
+            contentScrollView.topAnchor.constraint(equalTo: view.topAnchor),
+            contentScrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            contentScrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            contentScrollView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            
+            loadingView.leftAnchor.constraint(equalTo: contentScrollView.leftAnchor),
+            loadingView.rightAnchor.constraint(equalTo: contentScrollView.rightAnchor),
+            loadingView.bottomAnchor.constraint(equalTo: contentScrollView.bottomAnchor),
+            loadingView.topAnchor.constraint(equalTo: contentScrollView.topAnchor),
+        ])
+    }
+
+    private func configureMovieInfoListStackView(data: MovieInfo) {
+        let directors = data.directors.map { $0.name }.joined(separator: ", ")
```

Comment:
혹시 KeyPath에 대해 아실까요?

```suggestion
        let directors = data.directors.map(\.name)joined(separator: ", ")
```

Reply:
처음 보는 형태입니다. 키 경로를 사용하여 줄이는 방법도 있군요... 감사합니다 👍